src/client/view/container/container.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+import movie from './movie.json';
+import MovieList from '../movieList/movieList';
+import Search from '../search/search';
+import MovieCard from '../movieCard/movieCard';
+import BuggyButton from '../buggyButton/buggyButton';
+
+export default class Container extends React.Component {
+    constructor() {
```

Comment:
By the way, note that you can do it a little bit shorter:
`
state = data;
handleChangeSearchValue = (event) => {
        this.setState({searchValue: event.target.value});
    }
....
`
In this case you don't need constructor. That's kind of syntax sugar, but in my opinion looks nicer. Especially when there are a lot of functions to bind.

Reply:
I used this way